RecoEgamma/EgammaIsolationAlgos/python/egmPhotonIsolationAOD_cff.py

```diff
@@ -0,0 +1,40 @@
+import FWCore.ParameterSet.Config as cms
+
+from CommonTools.ParticleFlow.pfNoPileUpIso_cff import * 
+from CommonTools.ParticleFlow.pfParticleSelection_cff import * 
+
+
+pfNoPileUpCandidates = pfAllChargedHadrons.clone()
+pfNoPileUpCandidates.pdgId.extend(pfAllNeutralHadronsAndPhotons.pdgId)
+
+particleFlowTmpPtrs = cms.EDProducer("PFCandidateFwdPtrProducer",
+src = cms.InputTag('particleFlow')
+)
+
+egmPhotonIsolationAOD = cms.EDProducer( "CITKPFIsolationSumProducer",
+			  srcToIsolate = cms.InputTag("gedPhotons"),
+			  srcForIsolationCone = cms.InputTag('pfNoPileUpCandidates'),
+			  isolationConeDefinitions = cms.VPSet(
```

Comment:
isolationConeDefinitions is repeated in many places now.
Please define it once and use where needed.